pool/cqrs_command/mailing_command.ml

```diff
@@ -5,15 +5,31 @@ open Mailing
 
 let src = Logs.Src.create "mailing.cqrs"
 
-module Create : sig
-  include Common.CommandSig
+let default_command start_at end_at rate is_random distribution : update =
+  let distribution =
+    let open Distribution in
+    if is_random
+    then Some Random
+    else distribution |> CCOption.map (fun d -> Sorted d)
```

Comment:
use the available `create_sorted` function or use `ppx_variants_conv` for Distribution type, and then use `sorted`